pkg/schemas/security_integration_property_gen.go

```diff
@@ -0,0 +1,41 @@
+// Code generated by sdk-to-schema generator; DO NOT EDIT.
+
+package schemas
+
+import (
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// ShowSecurityIntegrationPropertySchema represents output of SHOW query for the single SecurityIntegrationProperty.
+var ShowSecurityIntegrationPropertySchema = map[string]*schema.Schema{
```

Comment:
Not Show, but Describe. Change in other places as well.

Reply:
It's explained in the generator comment